<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stalcraft Helper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="Input_item">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="aucList"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="Graph_time">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string>График цен</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Info">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string>Информация о предмете</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReset_graph"/>
    <addaction name="actionDelete_graph"/>
    <addaction name="actionSaveData"/>
    <addaction name="actionDelete_ALL_SavedData"/>
    <addaction name="actionDeleteDataHistoryLot"/>
    <addaction name="actionGetSavedLot"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionReset_graph">
   <property name="text">
    <string>Reset graph</string>
   </property>
  </action>
  <action name="actionDelete_graph">
   <property name="text">
    <string>Delete graph</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="text">
    <string>SaveData</string>
   </property>
  </action>
  <action name="actionDelete_ALL_SavedData">
   <property name="text">
    <string>Delete_ALL_SavedData</string>
   </property>
  </action>
  <action name="actionDeleteDataHistoryLot">
   <property name="text">
    <string>DeleteDataHistoryLot</string>
   </property>
  </action>
  <action name="actionGetSavedLot">
   <property name="text">
    <string>GetSavedLot</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
